operator/roles/kiali-deploy/templates/kubernetes/service.yaml

```
@@ -6,6 +6,9 @@ metadata:
   labels:
     app: kiali
     version: {{ kiali_vars.deployment.version_label }}
+  annotations:
+    {{ kiali_vars.apidocs.annotations.api_spec_annotation_name }}: {{ kiali_vars.server.web_root }}/swagger.json
```

If the web_root is just `/` then this causes a problem. But this is easily fixable by using an if-else expression:

```suggestion
    {{ kiali_vars.apidocs.annotations.api_spec_annotation_name }}: {{ '' if kiali_vars.server.web_root == '/' else kiali_vars.server.web_root}}/swagger.json
```

---

Thank you !